css/booklit.css

```diff
@@ -743,6 +743,7 @@ h3 .anchor {
 }
 .top-nav {
   position: relative;
+  white-space: nowrap;
```

Comment:
This change needs to be made to `less/booklit.less` instead and then compiled :slightly_smiling_face:

Reply:
Knew I'd miss something. I don't understand this newer css stuff :(

Reply:
I initially misunderstood your comment. Thought I had to _only_ make the change in the `/less` directory, but I had to make the change in both `less` and `css`. I needlessly deleted my commit haha.